src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -127,7 +127,8 @@ public async Task InitializeAsync()
                 await this.blockRepository.ReIndexAsync().ConfigureAwait(false);
             }
 
-            ChainedHeader initializationTip = this.chain.GetBlock(this.blockRepository.BlockHash);
+            // TODO ACTIVATION consider using height there
```

Comment:
Height wouldn't work here as you could get a block which is different

Reply:
yeah. I left those TODOs for the follow up PR to check where we will need those. Didn't check where we need them right now. 
But will delete for this one because you already checked